.idea/php.xml

```diff
@@ -95,7 +95,9 @@
       <path value="$PROJECT_DIR$/vendor/symfony/framework-bundle" />
     </include_path>
   </component>
-  <component name="PhpProjectSharedConfiguration" php_language_level="7.3" />
+  <component name="PhpProjectSharedConfiguration" php_language_level="7.3">
+    <option name="suggestChangeDefaultLanguageLevel" value="false" />
```

Comment:
dieser wert wurde von phpstorm automatisch geschrieben als ich "revert to previous language level" ausgewählt hab (nachdem phpstorm voll-automatisch gedacht hattes es müsste php8 auswählen)